src/vivarium_public_health/risks/data_transformations.py

```diff
@@ -409,23 +438,23 @@ def validate_distribution_data_source(builder, risk: EntityString):
         if not category_thresholds:
             raise ValueError("Must specify category thresholds to use alternative risks.")
 
-    elif risk.type in ["risk_factor", "coverage_gap"]:
-        if isinstance(exposure_type, (int, float)) and not 0 <= exposure_type <= 1:
-            raise ValueError(f"Exposure should be in the range [0, 1]")
-        elif isinstance(exposure_type, str) and exposure_type.split(".")[0] not in [
-            "covariate",
-            "data",
-        ]:
-            raise ValueError(
-                f"Exposure must be specified as 'data', an integer or float value, "
-                f"or as a string in the format covariate.covariate_name"
-            )
-        else:
-            pass  # All good
-    else:
+    elif risk.type not in ["risk_factor", "coverage_gap"]:
         raise ValueError(f"Unknown risk type {risk.type} for risk {risk.name}")
 
 
+def validate_distribution_data(distribution_data: Dict[str, Any]) -> None:
+    exposure_data = distribution_data["exposure"]
+    val_cols = distribution_data["exposure_value_columns"]
+    if distribution_data["distribution_type"] == "dichotomous":
+        if ((exposure_data[val_cols] < 0) | exposure_data[val_cols] > 1).any().any():
+            raise ValueError(f"Exposure should be in the range [0, 1]")
+    # TODO: validate that weights, standard deviation, and exposure have the
```

Comment:
Not in this epic I assume?

Reply:
We probably should do this while we're here, but after we're done with everything that is required for the MNCH model